File: core/src/main/java/io/github/keep2iron/pejoy/utilities/Platform.java
package io.github.keep2iron.pejoy.utilities;

import android.os.Build;

/**
 * @author JoongWon Baik
 */
public class Platform {
    public static boolean hasICS() {
        return Build.VERSION.SDK_INT >= Build.VERSION_CODES.ICE_CREAM_SANDWICH;
    }

    public static boolean hasKitKat() {
        return Build.VERSION.SDK_INT >= Build.VERSION_CODES.KITKAT;
    }
}

File: core/src/main/java/io/github/keep2iron/pejoy/utilities/Platform.kt
package io.github.keep2iron.pejoy.utilities

import android.os.Build

/**
 * @author JoongWon Baik
 */
object Platform {

  fun hasKitKat(): Boolean {
    return Build.VERSION.SDK_INT >= Build.VERSION_CODES.KITKAT
  }
}